Source/platform/locale.cpp

```diff
@@ -11,6 +11,9 @@
 #elif defined(__3DS__)
 #include "platform/ctr/locale.hpp"
 #elif defined(_WIN32)
+// Suppress definitions of `min` and `max` macros by <windows.h>:
+#define NOMINMAX 1

```

Comment:
This appears to already be there

Reply:
Rebase accident, thanks @ephphatha for fixing!